frameworks/elastic/tests/test_backup_restore.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+import os
+from typing import Iterator
+import sdk_install
+import sdk_cmd
+import sdk_tasks
+from tests import config
+
+
+@pytest.fixture(scope="module", autouse=True)
+def configure_package(configure_security: None) -> Iterator[None]:
+
+    try:
+        service_options = {"elasticsearch": {"plugins": "repository-s3"}}
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+        sdk_install.install(
+            config.PACKAGE_NAME,
+            config.SERVICE_NAME,
+            config.DEFAULT_TASK_COUNT,
+            additional_options=service_options,
+        )
+
+        yield  # let the test session execute
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.fixture(autouse=True)
+def pre_test_setup() -> None:
+    sdk_tasks.check_running(config.SERVICE_NAME, config.DEFAULT_TASK_COUNT)
+    config.wait_for_expected_nodes_to_exist(task_count=config.DEFAULT_TASK_COUNT)
+
+
+@pytest.mark.aws
+@pytest.mark.sanity
+def test_backup_restore():
+    key_id = os.getenv("AWS_ACCESS_KEY_ID")
+    if not key_id:
+        assert (
+            False
+        ), 'AWS credentials are required for this test. Disable test with e.g. TEST_TYPES="sanity and not aws"'
+    master_0_node_id = sdk_tasks.get_task_ids(config.SERVICE_NAME, "master-0-node")
+    task_list = sdk_tasks.get_task_ids(config.SERVICE_NAME)
+    for id in task_list:
+        sdk_cmd.run_cli(
```

Comment:
Could `sdk_cmd.service_task_exec` be used in all cases where `sdk_cmd.run_cli` is being used?

Reply:
Done